lib/backend/k8s/conversion/conversion.go

```diff
@@ -173,7 +173,7 @@ func (c Converter) PodToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 
 	// Generate the interface name based on workload.  This must match
 	// the host-side veth configured by the CNI plugin.
-	interfaceName := VethNameForWorkload(wepName)
+	interfaceName := VethNameForWorkload(fmt.Sprintf("%s/%s", wepName, pod.Namespace))
```

Comment:
If this is called from elsewhere and the result needs to be the same, then VethNameForWorkload should be updated to take namespace as a parameter and it should do the concatenation.

Reply:
This is only the case for k8s code path in CNI. For the non-k8s code path, we use `cali+first 11 digits of the ContainerID` for the vethname

Reply:
o ok I misread that comment, I'll change it